cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2190,6 +2190,40 @@ search = `cs_crowdstrike_eventstream` | eval crowdstrike_userid=coalesce('event.
 | eval _key=crowdstrike_userid | outputlookup cs_crowdstrike_inventory \
 | `cs_device_inventory_out_to_temp_lookup`
 
+[Device Inventory - Kaspersky]
+disabled = 0
+enableSched = 1
+alert.track = 0
+cron_schedule = 10 * * * *
+description = This report update the device inventory every hour and generates lookup for Kaspersky data. \
+\
+Data Collection - The Kaspersky data needs to be collected via the Kaspersky Add-on for Splunk (https://splunkbase.splunk.com/app/4656/#/details).
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_kaspersky` ProductName=KES*  \
+| rename dst as ip , dest as hostname,  ProductName as kaspersky_collected_by , ProductVersion as kaspersky_version, macaddress as mac_address  \
+| eval hostname=lower(hostname) \
+| table _time  ip, hostname, kaspersky_collected_by, kaspersky_version   \
+| fillnull ip,hostname,mac_address value=""  \
+| stats latest(_time) as _time latest(*) as * by hostname \
+| join type=left [| search `cs_kaspersky`  log_type=KLSRV_HOST_STATUS_CRITICAL "Protection is disabled" \
+    | rex field=_raw "Status of device \'(?<critical_device>[^\']*)" \
+    | stats latest(_time) as time  by critical_device \
+    | rename critical_device as hostname \
+    |  eval hostname=lower(hostname) ]  \
+| eval kaspersky_status=if(_time<time AND isnotnull(time),"Absent","Present")  | fields - time \
```

Comment:
Bro, the status should be "Protection Disabled" or "Disabled" instead of "Absent".
- So there will be 3 types on the Device Inventory: Green (Present or Enabled), Red (Protection Disabled or Disabled), Grey (Absent - for hosts which do not send Kaspersky related events or host on which Kaspersky is not installed)
- "Absent" will be calculated directly in the device inventory dashboard, for all the host for which Kaspersky lookup do not have any data